Cargo.toml

```diff
@@ -120,5 +121,5 @@ test-attestation-cert-ids = []
 # interchange = { git = "https://github.com/trussed-dev/interchange", branch = "main" }
 
 [package.metadata.docs.rs]
-features = ["virt"]
+features = ["ext", "virt"]
```

Comment:
I'm not even 100% what "virt" is. A `mock-platform`? As it's not just running in RAM, but also using the hardcoded seed of 42s. Probably we should decouple this anyway, so we have a builder with a real `CryptoCoreRng`, one passing a seed, and perhaps for ultimate unsafe convenience one with a hardcoded seed.

Reply:
The hardcoded seed dates back to this commit:
https://github.com/trussed-dev/trussed/commit/87e0c64ad94af978012cf9401bbeb5d65c29528e#diff-2df46fbc46e4818d054b9537e84ce0a18d7c95064657a2422040ac9d30e96914

I did not have the time and need to investigate this yet, but making this configurable would of course be better.